src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const allMen = people.filter((person) => person.sex === 'm');
+  const menFromThisCentury = century === undefined
```

Comment:
No need to compare the value with undefined, just check if it exists: `century ? ... : ...;`
Fix this in all code.